cardano-cli/src/Cardano/CLI/EraBased/Run/Governance.hs

```diff
@@ -54,27 +104,40 @@ runGovernanceMIRCertificatePayStakeAddrs w mirPot sAddrs rwdAmts oFp = do
     mirCertDesc :: TextEnvelopeDescr
     mirCertDesc = "Move Instantaneous Rewards Certificate"
 
-runGovernanceMIRCertificateTransfer
-  :: ShelleyToBabbageEra era
+runGovernanceCreateMirCertificateTransferToTreasuryCmd :: ()
+  => ShelleyToBabbageEra era
   -> Lovelace
   -> File () Out
-  -> TransferDirection
   -> ExceptT GovernanceCmdError IO ()
-runGovernanceMIRCertificateTransfer w ll oFp direction = do
+runGovernanceCreateMirCertificateTransferToTreasuryCmd w ll oFp = do
   let mirTarget = Ledger.SendToOppositePotMIR (toShelleyLovelace ll)
 
-  let mirCert =
-        makeMIRCertificate $
-          case direction of
-            TransferToReserves -> MirCertificateRequirements w Ledger.TreasuryMIR mirTarget
-            TransferToTreasury -> MirCertificateRequirements w Ledger.ReservesMIR mirTarget
```

Comment:
I don't understand why reserves are matched to treasury and vice versa.

Reply:
Last two arguments are pot and target. The pot is the source I guess. This would mean: from treasury and from reserves as far as I understand.

Reply:
It was confusingly named in ledger so `TransferDirection` was created to make it more obvious what was happening.